apps/condo/lang/ru/ru.json

```
@@ -3478,5 +3464,19 @@
   "ticket.problem.classifier.PEST_CONTROL_DISINFECTION.name": "Дератизация, дезинсекция, дезинфекция",
   "ticket.problem.classifier.CRACKS_MASONRY_PLASTER_DAMAGE.name": "Трещины, выпадение кладки/штукатурки, повреждение отмостки/цоколя",
   "ticket.problem.classifier.SIGNAGE_ADVERTISING_EQUIPMENT_ON_FACADE.name": "Надпись/реклама/оборудование на фасаде",
-  "ticket.problem.classifier.REPAIR_IN_ROOM.name": "Ремонт в помещении"
+  "ticket.problem.classifier.REPAIR_IN_ROOM.name": "Ремонт в помещении",
+  "ticket.place.classifier.IT_SERVICES.name": "IT сервисы",
+  "ticket.place.classifier.COURTYARD.name": "Двор",
+  "ticket.place.classifier.APARTMENT.name": "Квартира",
+  "ticket.place.classifier.RECEIPT.name": "Квитанция",
+  "ticket.place.classifier.ENTRANCEWAY.name": "Подъезд",
```

```suggestion
  "ticket.place.classifier.SECTION.name": "Подъезд",
```

to be aligned with our PropertyMap